src/Sulu/Bundle/MediaBundle/Tests/Unit/Media/ImageConverter/MediaImageExtractorTest.php

```diff
@@ -50,24 +50,6 @@ public function setUp(): void
         );
     }
 
-    public function testExtractUnseekableResource(): void
-    {
-        $resource = $this->createUnseekableResource();
-
-        // this test just check if "createUnseekableResource" will produce the correct unseekable resource to reproduce
-        // bug with aws s3 unseekable resources: https://github.com/sulu/sulu/issues/5468
-        // if this test fail the createUnseekableResource is maybe wrong
-        $this->expectWarning();
-
-        if (\PHP_VERSION_ID >= 80000) {
-            $this->expectWarningMessage('mime_content_type(): Stream does not support seeking');
-        } else {
-            $this->expectWarningMessage('mime_content_type(): stream does not support seeking');
-        }
-
-        $this->mediaImageExtractor->extract($resource);
-    }
```

Comment:
The `createUnseekableResource` make sure its unseekable resource so we need not test if the warning is thrown.